website/docs/advanced/strategy-constraints.md

```diff
@@ -5,52 +5,169 @@ title: Strategy Constraints
 
 :::info Availability
 Strategy constraints are available to Unleash Pro and Enterprise users.
+
+Unleash 4.9 introduced a more comprehensive set of constraint operators. These require that both Unleash *and* your client SDK of choice support them. See the [SDK compatibility table](../sdks/index.md#server-side-compatibility-table) for more information.
 :::
 
 :::tip
 This page explains what strategy constraints are in Unleash and how they work. If you want to know *how you add* strategy constraints to an activation strategy, see [the corresponding how-to guide](../how-to/how-to-add-strategy-constraints.md "how to add strategy constraints").
 :::
 
-**Strategy constraints** allow you to set preconditions on activation strategies that must be satisfied for the activation strategy to take effect. For example, you might want a strategy to only trigger if a user belongs to a specific group or is in a specific country.
+**Strategy constraints** are conditions that must be satisifed for an [activation strategy](../user_guide/activation_strategy) to be evaluated for a feature toggle.
+With strategy constraints, you can:
+- roll out a feature **only to users in a specific region**
+- schedule a feature to be **released at a specific time**
+- make a feature available for **a limited time only**
+- release a feature to users with one of a set of **email addresses**
+- ... and much more!
+
+
+Strategy constraints use fields from the [Unleash Context](../user_guide/unleash_context) to determine whether a strategy should apply or not.
+You can constrain both on [standard context fields](../user_guide/unleash_context#structure) and on [custom context fields](../user_guide/unleash_context#custom-context-fields).
+
+## Constraining on custom context fields
+
+:::info Making custom context fields available
+To be able to constrain on a field, it must be listed under the Context Field menu. If a field isn't listed, you can add it yourself. See [the how-to guide for creating your own custom fields](../how-to/how-to-define-custom-context-fields.md) for more info.
+:::
 
-Constraints use fields from the [Unleash Context](../user_guide/unleash_context) to determine whether a strategy should apply or not. You can constrain on both [standard context fields](../user_guide/unleash_context#structure) and on [custom context fields](../user_guide/unleash_context#custom-context-fields). A common use for using custom context fields is a multi-tenant service where you want to use a tenant identifier to control the feature rollout. This would allow you to decide which users should get access to your new feature based on the tenant. Other commonly seen custom context fields include fields for region, country, and customer type.
+Unleash only provides a limited set of context fields by default, and they may not fulfill all your needs.
```

Comment:
@FredrikOseberg With the upcoming segmentation feature, it seems that we'll bundle in a few custom fields by default, such as "region" etc. Does that mean these are promoted to standard field status, or are they just "preconfigured custom fields"?

Reply:
Not certain what we'll call it. Open for discussion! Standard does have a nice ring to it though.

Reply:
Got it! I'll leave that discussion for later. Thanks!